new-site/src/components/PageTabs.js

```diff
@@ -0,0 +1,80 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { navigate } from 'gatsby';
+import { Tabs } from 'hds-react';
+
+import './PageTabs.scss';
+
+const resolvePathFromSlug = (slug) => {
+  const parts = slug
+    .trim()
+    .split('/')
+    .filter((p) => !!p);
+  return parts.slice(0, -1).join('/');
+};
+
+const PageTabs = ({ pageContext, children }) => {
+  const slug = pageContext.frontmatter.slug;
+  const mdxChildren = React.Children.toArray(children);
+  const tabList = mdxChildren.find((reactChild) => reactChild.type.componentName === 'PageTabList');
+  const tabPanel = mdxChildren.find((reactChild) => reactChild.type.componentName === 'PageTabPanel');
+  const tabs = tabList.props?.children.filter((reactChild) => reactChild.type.componentName === 'PageTab');
+  const activeIndex = tabs.findIndex((tab) => slug.endsWith(tab.props.href));
+  const basePath = resolvePathFromSlug(slug, activeIndex === 0);
```

Comment:
The resolvePathFromSlug function takes only one argument

Reply:
- [x] remove unused argument